src/saml2/mdstore.py

```diff
@@ -1290,6 +1301,32 @@ def attribute_requirement(self, entity_id, index=None):
             if entity_id in _md:
                 return _md.attribute_requirement(entity_id, index)
 
+    def subject_id_requirement(self, entity_id):
+        try:
+            entity_attributes = self.entity_attributes(entity_id)
+        except KeyError:
+            return None
+
+        if "urn:oasis:names:tc:SAML:profiles:subject-id:req" in entity_attributes:
+            subject_id_req = entity_attributes["urn:oasis:names:tc:SAML:profiles:subject-id:req"][0]
+            if subject_id_req == "any" or subject_id_req == "pairwise-id":
```

Comment:
This means that "any" results to a pairwise identifier. I don't mind this, but it is also good to mention that this choice was made here. This should be mentioned in the changelog.